content/shmem_team_broadcast.tex

```diff
@@ -0,0 +1,90 @@
+\apisummary{
+    Broadcasts a block of data from one \ac{PE} to one or more destination
+    \acp{PE}.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void @\FuncDecl{shmem\_team\_broadcast32}@(shmem_team_t team, void *dest, const void *source, size_t nelems, int PE_root);
```

Comment:
@gmegan Likely this has been discussed already but I missed it. Why is `psync` missing from the argument list? Is it part of  the struct `shmem_team_t`?

Reply:
Yes. In this version of the API, we are asking that the library implementor provide and manage any required communication structures associated with the given team.